app/editor/src/features/content/form/PropertiesContentForm.tsx

```diff
@@ -71,18 +65,18 @@ export const PropertiesContentForm: React.FC<IContentSubForms> = ({ setContent,
                 className="md-lrg"
                 name="date"
                 label="Date"
-                selectedDate={values.createdOn ?? ''}
-                value={values?.createdOn}
+                selectedDate={values.publishedOn ?? ''}
+                value={values.publishedOn}
                 disabled
                 onChange={(date: any) => {
-                  setContent({ ...content, createdOn: date });
+                  setContent({ ...content, publishedOn: date });
                 }}
               />
             </Col>
             <Col style={{ marginTop: '0.45em' }}>
               <FormikText
                 disabled
-                value={formatTime(values?.createdOn!)}
+                value={values.publishedOn ? formatTime(values.publishedOn!) : ''}
```

Comment:
This time thing will not work.  Not sure if you're already resolving this.

Reply:
Yup am resolving this!